package.json

```diff
@@ -17,10 +17,14 @@
     "typescript": "^4.1.2",
     "web-vitals": "^1.0.1",
     "react-bootstrap": "^2.1.1",
-    "react-bootstrap-icons": "^1.7.2",
     "sass": "^1.49.0",
```

Comment:
We cannot totally get rid of the bootstrap icons since we use them for the social media icons and elsewhere in the repo. In you want to go ahead with totally replacing bootstrap icons with font awesome, then please do it all over the repo.

Reply:
Oh sorry. I will replace the social icons to FA and re-add the bootstrap icons.